app/pathogen/sarscov2/layout.tsx

```diff
@@ -1,43 +1,48 @@
-import React from "react";
```

Comment:
If you're wondering, when the flag isn't specified, `process.env.SARS_COV_2_TRACKER_ENABLED` is `undefined` which is falsy so this will have the right value even if we don't change anything in vercel hahaha